src/factories/IProposalFactory.sol

```diff
@@ -28,6 +31,7 @@ interface IProposalFactory {
     struct ModuleConfig {
         IModule.Metadata metadata;
         bytes configdata;
+        bytes dependencydata;
```

Comment:
Why are we not using camelCase here? I see that you just copied the style that already existed, so just as a general q.

Reply:
I would agree to change that.
Lets move it to #258 
Otherwise this PR gets a bit bloated

Reply:
I agree, we can fix things like this in #258!